templates/openwisp2/settings.py

```diff
@@ -162,9 +183,45 @@
 CELERY_BEAT_SCHEDULE = {
     'delete_old_notifications': {
         'task': 'openwisp_notifications.tasks.delete_old_notifications',
-        'schedule': timedelta(days=1),
+        'schedule': crontab(**{ {{ cron_delete_old_notifications }} }),
         'args': ({{ openwisp2_notifications_delete_old_notifications }},),
     },
+{% if openwisp2_radius %}
+
+    'deactivate_expired_users': {
+        'task': 'openwisp_radius.tasks.cleanup_stale_radacct',
```

Comment:
this looks wrong, since `openwisp_radius.tasks.cleanup_stale_radacct` is repeated below